pkg/serverless/invocationlifecycle/lifecycle_test.go

```diff
@@ -1069,3 +1057,7 @@ func getEventFromFile(filename string) []byte {
 	buf.WriteString("0")
 	return buf.Bytes()
 }
+
+func createDemultiplexer(t *testing.T) demultiplexer.FakeSamplerMock {
+	return fxutil.Test[demultiplexer.FakeSamplerMock](t, logimpl.MockModule(), demultiplexerimpl.FakeSamplerMockModule())
+}
```

Comment:
We are duplicating this function in different test files. Should we try moving to a shared place so that if in the future we need to change, we do not have to change in multiple places?

Reply:
Agreed

Reply:
Agreed. I considered it but didn't do it because:
- I cannot find a good location
- This may evolves differently based on test.

Do you have a suggestion where to put this function?

Reply:
In `comp/aggregator/demultiplexer/component_mock.go` maybe ?
Or a `test_helper.go` file in the same package ?

Reply:
I think both of the suggestion from @pgimalac could be a good place to add that test helper 😄